README.md

```diff
@@ -775,6 +774,13 @@ User can add a STM32 based board following this [wiki](https://github.com/stm32d
 | :----: | :-------: | ---- | :-----: | :---- |
 | :green_heart:  | STM32WB55CE | [SharkyMKR](https://midatronics.com/shop/development-boards/mkr-sharky-i/) | *1.7.0* |  |
 
+### [SparkFun](https://www.sparkfun.com/) boards
+
+| Status | Device(s) | Name | Release | Notes |
+| :----: | :-------: | ---- | :-----: | :---- |
+| :yellow_heart: | STM32F405RG | [SparkFun MicroMod Processor Board - STM32F405](https://www.sparkfun.com/products/21326) | **2.6.0** |  |
+| :yellow_heart: | STM32WB5MMG | [SparkFun MicroMod Processor Board - STM32WB5MMG](https://www.sparkfun.com/products/21438) | **2.6.0** |  |
```

Comment:
```suggestion
| :yellow_heart: | STM32WB5MMG | SparkFun MicroMod Processor Board - STM32WB5MMG | **2.6.0** |  |
```

Reply:
I'll add the link back in when it's live. Thanks!